nrfx_glue.h

```diff
@@ -257,8 +257,14 @@ extern const uint32_t z_bt_ctlr_used_nrf_ppi_groups;
 
 #if defined(CONFIG_PWM_NRF5_SW)
 #define PWM_NRF5_SW_NODE DT_INST(0, nordic_nrf_sw_pwm)
+#if DT_NODE_HAS_PROP(PWM_NRF5_SW_NODE, rtc_instance)
+#define PWM_NRF5_SW_NUM_CHANNELS_USED	3
+#else /* DT_NODE_HAS_PROP(PWM_NRF5_SW_NODE, rtc_instance) */
+#define PWM_NRF5_SW_NUM_CHANNELS_USED	2
+#endif /* DT_NODE_HAS_PROP(PWM_NRF5_SW_NODE, rtc_instance) */
 #define NRFX_PPI_CHANNELS_USED_BY_PWM_SW \
-    (BIT_MASK(DT_PROP(PWM_NRF5_SW_NODE, channel_count) * 2) \
+    (BIT_MASK(DT_PROP(PWM_NRF5_SW_NODE, channel_count) *	\
+	      PWM_NRF5_SW_NUM_CHANNELS_USED)			\
```

Comment:
This name is a bit confusing, as the `channel_count` right above refers to a different kind of channels. How about `PWM_NRF5_SW_PPI_CHANNELS_PER_PIN`?

Reply:
Sounds good! Fixed.